### PR TITLE
Remove remaining legacy plexus components

### PR DESCRIPTION
**Pull request by kwin**
This relates to #206

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by kwin** [comment]
@cziegeler Can you review this?

File: aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/AbstractAemMojo.java [hunk 1]
```diff
@@ -34,6 +34,7 @@
 import org.eclipse.aether.resolution.ArtifactResolutionException;
 import org.eclipse.aether.resolution.ArtifactResult;
 
+import com.adobe.aem.analyser.mojos.extensions.DependencyLifecycleParticipant;
```
Comment (cziegeler) [inline aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/AbstractAemMojo.java:37]: Why is this import needed?

File: aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/extensions/DependencyLifecycleParticipant.java [hunk 2]
```diff
@@ -9,7 +9,7 @@
   OF ANY KIND, either express or implied. See the License for the specific language
   governing permissions and limitations under the License.
 */
-package com.adobe.aem.analyser.mojos;
+package com.adobe.aem.analyser.mojos.extensions;
```
Comment (cziegeler) [inline aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/extensions/DependencyLifecycleParticipant.java:12]: Why are those imports needed?

File: aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/AbstractAemMojo.java (hunk 1 above)
Comment (kwin) [reply aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/AbstractAemMojo.java:37]: Because I moved all extensions to a dedicated subpackage `com.adobe.aem.analyser.mojos.extensions`. It used to be in the same package as this Mojo before and therefore no import was necessary.

File: aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/extensions/DependencyLifecycleParticipant.java (hunk 2 above)
Comment (kwin) [reply aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/extensions/DependencyLifecycleParticipant.java:12]: See above

Comment (cziegeler) [reply aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/extensions/DependencyLifecycleParticipant.java:12]: sorry, all good - didnt had my glasses on :)

**Review by cziegeler — APPROVED** [review]
Thanks @kwin - looks good to me. I'll merge after the next release of the plugin which should happen shortly
